docs/en/writing-running-appium/caps.md

```diff
@@ -29,6 +29,7 @@
 |`appPackage`| Java package of the Android app you want to run|`com.example.android.myApp`, `com.android.settings`|
 |`appWaitActivity`| Activity name for the Android activity you want to wait for|`SplashActivity`|
 |`appWaitPackage`| Java package of the Android app you want to wait for|`com.example.android.myApp`, `com.android.settings`|
+|`appWaitDuration`| Timeout in milliseconds used to wait for the appWaitActivity to launch (default `20000`), `30000`|
```

Comment:
This needs a `|` instead of `,` before `3000`. Otherwise it gets in the wrong column in the table.